oak_tensorflow_service/src/tflite.rs

```diff
@@ -107,13 +108,16 @@ impl TfliteModel {
                 &mut output_buffer_len,
             )
         };
-        self.output_buffer_len = Some(output_buffer_len);
-
-        log!(
-            Level::Info,
-            "TFLite model has been successfully initialized"
-        );
-        Ok(())
+        if result_code == 0 {
+            self.output_buffer_len = Some(output_buffer_len);
+            log!(
+                Level::Info,
+                "TFLite model has been successfully initialized"
+            );
+            Ok(())
+        } else {
+            Err(anyhow!("Failed to initialize TFLite model, code: {}", result_code))
```

Comment:
Please use consistent capitalization for new errors, see #3467

Reply:
Done